teuthology/suite/run.py

```diff
@@ -505,7 +505,9 @@ def schedule_jobs(self, jobs_missing_packages, jobs_to_schedule, name):
 
     def check_priority(self, jobs_to_schedule):
         priority = self.args.priority
-        msg='''Use the following testing priority
+        msg=f'''Error: Unable to schedule {jobs_to_schedule} jobs with priority {priority}.
```

Comment:
Because this message goes to exception it is not needed to be prefixed with "Error: ", please remove it.

Reply:
done